stubs/cryptography/cryptography/utils.pyi

```diff
@@ -0,0 +1,38 @@
+import enum
+import typing
+from typing import Any
+
+class CryptographyDeprecationWarning(UserWarning): ...
+
+PersistentlyDeprecated2017 = CryptographyDeprecationWarning
+PersistentlyDeprecated2019 = CryptographyDeprecationWarning
+DeprecatedIn34 = CryptographyDeprecationWarning
+
+def read_only_property(name: str) -> Any: ...
+def register_interface(iface: object): ...
```

Comment:
```suggestion
def register_interface(iface: Any): ...
```

As not any arbitrary object can be used as an interface. Affects more functions below.

Reply:
> Affects more functions below.

I'm not sure what you mean by this. Can you please clarify?

Reply:
Can you please clarify what is the difference between using `Any` and `object`? In what cases should I use one or the other?

Reply:
`object` should only be rarely used. In arguments only if an argument accepts literally anything (see my example below using `my_str`). `Any` should be used in cases where the argument can't be expressed properly using the current type system. In all other cases, where you can't determine the argument type, just leave it unannotated.

Reply:
```suggestion
def register_interface(iface: abc.ABCMeta): ...
```

Reply:
the type annotation is now:

```python
from typing_extensions import Protocol

_T_class = typing.TypeVar("_T_class", bound=type)

class _RegisterDecoratorType(Protocol):
    def __call__(
        self, klass: _T_class, *, check_annotations: bool = False
    ) -> _T_class:
        ...


def register_interface(iface: abc.ABCMeta) -> _RegisterDecoratorType: ...
```